Dockerfile-debian.template

```
@@ -39,25 +40,31 @@ RUN set -x \
 # see https://github.com/docker-library/haproxy/issues/94#issuecomment-505673353 for more details about prometheus support
 			contrib/prometheus-exporter/service-prometheus.o \
 		" \
-	' \
-	&& nproc="$(nproc)" \
-	&& eval "make -C /usr/src/haproxy -j '$nproc' all $makeOpts" \
-	&& eval "make -C /usr/src/haproxy install-bin $makeOpts" \
+	'; \
+# https://salsa.debian.org/haproxy-team/haproxy/-/commit/53988af3d006ebcbf2c941e34121859fd6379c70
+	case "$(dpkg --print-architecture)" in \
```

:eyes: This should be split out in case `dpkg --print-architecture` fails for some obscure reason:

```suggestion
	dpkgArch="$(dpkg --print-architecture)"; \
	case "$dpkgArch" in \
```